src/index.js

```diff
@@ -50,26 +101,42 @@ const uploadSwaggerToS3 = async function uploadSwaggerToS3(swagger, bucket, key,
   }).promise()
 }
 
-const getBucketAndKey = function getBucketAndKey(serverless) {
+const getServerlessOptions = function getServerlessOptions(serverless) {
   const bucket = serverless.service.custom.swaggerDestinations.s3BucketName
   const key = serverless.service.custom.swaggerDestinations.s3KeyName
+  const openapi = serverless.service.custom.swaggerDestinations.uploadOpenAPI
+  const yaml = serverless.service.custom.swaggerDestinations.uploadYaml
   if (!bucket || !key) {
     throw new Error('ExportSwagger: Bucket name and key are required fields')
   }
-  return { bucket, key }
-};
+  return { bucket, key, openapi, yaml }
+}
 
 const exportApi = async function exportApi(serverless) {
   const provider = serverless.getProvider('aws')
   const awsCredentials = provider.getCredentials().credentials
   const region = provider.getRegion()
   const stage = provider.getStage()
   const serviceName = serverless.service.getServiceName()
-  const { bucket, key } = getBucketAndKey(serverless)
+  const { bucket, key, openapi, yaml } = getServerlessOptions(serverless)
   const apiName = await getApiName(serviceName, stage, region, awsCredentials)
   const swagger = await getSwagger(apiName, stage, region, awsCredentials)
-  await uploadSwaggerToS3(swagger, bucket, key, region, awsCredentials)
-  serverless.cli.consoleLog('ExportSwagger: File uploaded to s3.')
+  const swaggeryaml = await getSwaggerYaml(apiName, stage, region, awsCredentials)
+  const oas30 = await getOpenapi(apiName, stage, region, awsCredentials)
+  const oas30yaml = await getOpenapiYaml(apiName, stage, region, awsCredentials)
+  const swaggerKey = `${key}-swagger.json`
+  await uploadSwaggerToS3(swagger, bucket, swaggerKey, region, awsCredentials)
+  if (typeof openapi !== 'undefined') {
```

Comment:
rather than checking for ` !== 'undefined'` can you change this to ` === true`.. do this everywhere you are checking